parser/tests/PrintAndParse_test.go

```diff
@@ -0,0 +1,14 @@
+package tests
+
+import (
+	"testing"
+	"twittertracker/parser"
+)
+
+func Test_ParseString(t *testing.T) {
+	// assert := assert.New(t)
```

Comment:
why is this commented out?

Reply:
We should still maintain this unit test. If you don’t want to I can fix it in another pr.